modules/configuration-resource-overview.adoc

```diff
@@ -26,7 +26,7 @@ You can customize the following Configuration Resources:
 |
 
 |Samples
-| * *ManagementState:*
+| * *managementState:*
 ** *Managed.* The operator updates the samples as the configuration dictates.
 ** *Unmanaged.* The operator ignores updates to the samples resource object and
 any imagestreams or templates in the OpenShift namespace.
```

Comment:
@IronicBadger To keep consistent, other options need updated too in this doc.
s/Samples Registry/samplesRegistry
s/Architecture/architecture
s/Skipped Imagestreams/skippedImagestreams
s/Skipped Templates/skippedTemplates